src/BloomBrowserUI/bookEdit/BookPagesThumbnailList/BookPagesThumbnailList.htm

```diff
@@ -61,7 +65,7 @@
             // This delay is needed because the processing of larger images was causing out-of-memory errors if several
             // images were requested before the server had finished processing previous images requests. We are also
 			// attempting to prevent the browser from timing out and showing the alt text instead of the image.
```

Comment:
Do we need to add to this comment that now the delay is also being used to make the initial Edit Tab load more responsive?